example/spec/parallel/testAPI/oneTimeRule.js

```diff
@@ -126,22 +127,35 @@ describe('When I create a one-time rule via the Cumulus API', () => {
   });
 
   describe('When listing the rules via the API', () => {
-    let listRulesResponse = '';
+    let listRules = '';
 
     beforeAll(async () => {
-      listRulesResponse = await rulesApiTestUtils.listRules({
+      const listRulesResponse = await rulesApiTestUtils.listRules({
         prefix: config.stackName
       });
 
-      console.log(JSON.stringify(listRulesResponse));
+      listRules = JSON.parse(listRulesResponse.body);
+      console.log(JSON.stringify(listRules));
```

Comment:
Are we intentionally including the console.log?

Reply:
Not sure, It was there before so I kept it, but I can remove it

Reply:
It can be removed.